test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpClientTests.cs

```diff
@@ -0,0 +1,66 @@
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    public class HttpClientTests : TestHelper
```

Comment:
It would be nice to have tests for the case where tracing is disabled on the request too.

Reply:
Added in https://github.com/DataDog/dd-trace-dotnet/pull/267/commits/7c1874cda63c1e61d61acdb757e544ebf4050d2d